src/components/common/GallerySlider/GallerySlider.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styles from './GallerySlider.module.scss';
+import { useSelector } from 'react-redux';
+import { getAll } from '../../../redux/productsRedux';
+import PanelLeftButton from './PanelLeftButton/PanelLeftButton';
+import Slider from './Slider/Slider';
+import Badge from './Badge/Badge';
+
+const GallerySlider = () => {
+  const topSeller = useSelector(getAll);
+
+  return (
+    <div className={styles.root}>
+      <div className={styles.topBtn}>
+        <a href='#'>Featured</a>
+        <a href='#' className={styles.active}>
+          Top seller
+        </a>
+        <a href='#'>Sale off</a>
+        <a href='#'>Top rated</a>
+      </div>
+      <PanelLeftButton />
+      <img
+        className={styles.image}
+        src={`${process.env.PUBLIC_URL}/images/products/${topSeller[2].image}`}
+        alt='chair'
+      />
+      <div className={styles.promo}>
+        <h4>${`${topSeller[2].price}`}</h4>
+        <h6>${`${topSeller[2].priceOld}`}</h6>
```

Comment:
Może warto by było renderować ten <h6> warunkowo w zależności czy priceOld w ogóle istnieje? Teraz jeśli wybierzemy tam produkt, który nie posiada priceOld to mamy przekreślony napis undefined ;)

Reply:
w sumie tak , dużo lepiej to będzie wyglądało :) poprawione , dodałem operator warunkowy na h6

Reply:
Spoko ;) Dla uproszenia kodu na przyszłość jak warunkowo coś renderujesz w stylu, że coś ma być lub czegoś ma nie być to możesz zrobić tak {topSeller[2].priceOld && <h6>${`${topSeller[2].priceOld}`}</h6>} , odejdzie Ci ten ELSE po ":" z którego i tak musisz zrobić nulla ;)